api/views/student.py

```diff
@@ -112,6 +112,23 @@ def missed_class(self, request: Request, personal_info_id: int) -> Response:  #
         )
         return Response(status=status.HTTP_204_NO_CONTENT)
 
+    @extend_schema(
+        responses={
+            status.HTTP_204_NO_CONTENT: OpenApiResponse(description="Action is taken"),
+            status.HTTP_400_BAD_REQUEST: OpenApiResponse(
+                response=ValidationErrorSerializer,
+                description="Something is wrong with the query params",
+            ),
+        },
+    )
```

Comment:
There could be a case for `CONFLICT` error when a coordinator for some reason sends this request for a student/teacher that is already not in the project anymore (i.e. already has status `LEFT_PREMATURELY` or `BANNED` + `FINISHED_LEFT` and `FINISHED_STAYS` for teacher and `FINISHED` for student).

But It's actually not a big deal and should be very rare, so maybe we shouldn't care, at least not in MVP.

Reply:
Probably yes, even if it happens it will just create a new log event